bin/riemann-elasticsearch

```diff
@@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
   end
 
   def health_url
-    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
+    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
```

Comment:
Well,
We could remove any leading and trailing slash, and put it there ourselves,
so we get consistent behavior?
On May 31, 2015 4:04 AM, "Kyle Kingsbury" <notifications@github.com> wrote:

> In bin/riemann-elasticsearch
> <https://github.com/aphyr/riemann-tools/pull/115#discussion_r31386100>:
>
> > @@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
> >    end
> >
> >    def health_url
> > -    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
> > +    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
>
> Er, well, if we remove one, then... do we enforce path prefixes contain a
> trailing slash, or?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aphyr/riemann-tools/pull/115/files#r31386100>.
>